docker-entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+set -e
+
+until psql $DATABASE_URL -c '\l' >/dev/null 2>&1; do
+    >&2 echo "Postgres is unavailable - sleeping"
+    sleep 1
+done
+
+>&2 echo "Postgres is up - continuing"
+
```

Comment:
It's probably worth running `migrate` here (at least optionally, if a certain env var is set). It's useful both locally and in deployed environments.

Reply:
The fabfile has a "migrate" task that uses "docker-compose run" to run the migrations.

I don't know how we'll want to run these when we get to kubernetes, but this seems convenient for running locally in docker for now.